.clang-format

```diff
@@ -1,6 +1,6 @@
 ---
 BasedOnStyle: Microsoft
-UseTab: AlignWithSpaces
+UseTab: ForIndentation
```

Comment:
@tigercosmos I think we should change this, it looks nicer, what do you think?
If you agree I can open a PR for Common++ as well

Reply:
I don't have a strong opinion about it. it's fine to me.

Reply:
ok, I'll open a separate PR for this change